ludwig/datasets/configs/adult_census_income.yaml

```diff
@@ -28,3 +28,4 @@ loader: adult_census_income.AdultCensusIncomeLoader
 description: |
   Predict whether income exceeds $50K/yr based on census data
   https://archive.ics.uci.edu/ml/datasets/adult
+task: Binary Classification
```

Comment:
Seem odd to specify the task in this way. In Ludwig, it would seem more appropriate to just list the output feature types, which is more aligned to how Ludwig thinks about tasks. Something like:

```
output_features:
  - name: <whatever>
     type: binary
```

Reply:
Good call, I'll switch over to this method since it conveys the info more clearly!